shell/platform/darwin/macos/framework/Source/MacOSSwitchableGLContext.h

```diff
@@ -0,0 +1,21 @@
+#include "flutter/flow/gl_context_switch.h"
+#include "flutter/fml/memory/thread_checker.h"
+
+#import <Cocoa/Cocoa.h>
+
+class MacOSSwitchableGLContext final : public flutter::SwitchableGLContext {
+ public:
+  explicit MacOSSwitchableGLContext(NSOpenGLContext* context);
+
+  bool SetCurrent() override;
```

Comment:
Please put the class name this is coming from in a comment, as is done throughout the engine codebase.